shared_model/cryptography/blob.hpp

```diff
@@ -81,8 +81,13 @@ namespace shared_model {
        * Design note: this method is deprecated and should be removed after
        * migration to shared model in whole project
        */
+
       template <typename BlobType>
-      [[deprecated]] BlobType makeOldModel() const {
+#ifndef SWIG
+      [[deprecated]]
+#endif
```

Comment:
better to define a separate file `deprecated.hpp` with 
```C++
#ifndef SWIG
#define DEPRECATED [[deprecated]]
#else
#define DEPRECATED
#endif
```

and then use only DEPRECATED keyword